edx/analytics/tasks/util/elasticsearch_target.py

```diff
@@ -26,32 +26,25 @@ class ElasticsearchTarget(HdfsTarget):
     Arguments:
             client (elasticsearch.Elasticsearch): An elasticsearch client.
             index (str): Name of the index that is populated.
-            doc_type (str): The doc_type that is written in the index.
             update_id (str): A unique identifier that is used to determine if an indexing task should be re-run.
     """
 
-    def __init__(self, client, index, doc_type, update_id):
+    def __init__(self, client, index, update_id):
         super(ElasticsearchTarget, self).__init__(is_tmp=True)
 
         self.marker_index = luigi.configuration.get_config().get(
             'elasticsearch',
             'marker-index',
             'index_updates'
         )
-        self.marker_doc_type = luigi.configuration.get_config().get(
-            'elasticsearch',
-            'marker-doc-type',
-            'marker'
-        )
         self.index = index
-        self.doc_type = doc_type
         self.update_id = update_id
 
         self.elasticsearch_client = client
 
     def marker_index_document_id(self):
         """A concise string that represents a unique ID for this instance of this task."""
-        params = '%s:%s:%s' % (self.index, self.doc_type, self.update_id)
+        params = '%s:%s' % (self.index, self.update_id)
```

Comment:
Ohh. We are in python3 now. Format strings are here or in the end `format`. Up to u.

Reply:
this service use python 2